files/image_config/rsyslog/rsyslog.conf.j2

```diff
@@ -72,3 +68,26 @@ $RepeatedMsgReduction on
 ###############
 #### RULES ####
 ###############
+
+#
+# Remote syslog logging
+#
+
+# The omfwd plug-in provides the core functionality of traditional message forwarding via UDP and plain TCP.
+# It is a built-in module that does not need to be loaded.
+
+{% if SYSLOG_SERVER is defined %}
+{% for server, data in SYSLOG_SERVER.items() %}
+{% set params_list = [] %}
+{% if 'source' in data %}
```

Comment:
>source [](http://example.com/codeflow?start=7&length=6)

Please add unit test.

Reply:
@qiluo-msft can you please clarify which UTs are you referring to? Do we have a Framework?

Reply:
Either sonic-config-engine unit test or sonic-mgmt test case.

Reply:
@qiluo-msft can we converge on this: https://github.com/Azure/sonic-mgmt/pull/5904 ?

Reply:
@qiluo-msft just a kind reminder